scripts/docker/Dockerfile

```diff
@@ -14,7 +14,7 @@ WORKDIR /go/src/github.com/hashicorp/vault
 COPY . .
 
 RUN make bootstrap \
-  && CGO_ENABLED=$CGO_ENABLED BUILD_TAGS="${BUILD_TAGS}" VAULT_DEV_BUILD=1 sh -c "'./scripts/build.sh'"
+  && CGO_ENABLED=$CGO_ENABLED BUILD_TAGS="${BUILD_TAGS}" OPENBAO_DEV_BUILD=1 sh -c "'./scripts/build.sh'"
 
 # Docker Image
 
```

Comment:
I think this needs more changes below as well, copying from `bin/bao` than `bin/vault`.

Reply:
fixed

Reply:
Thank you!